cocktail.php

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html lang="en">
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Cocktail Hour</title>
+    <link rel="stylesheet" href="normalize.css">
+    <link rel="stylesheet" href="style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Pacifico&display=swap" rel="stylesheet">
+</head>
+  <body>
+      <nav>
+        <div class="home"><a href="index.php">Home</a></div>
+        <div class="add"><a href="cocktail.php">Add A Cocktail</a></div>
+        <div class="search"><a href="#">Search</a></div>
+      </nav>
+    <h1>Cocktail Hour!</h1>
+
+    <?php 
```

Comment:
I'd add this at the top of your file - for convention! - we usually include or require files first thing in a file 😄